f2/src/utils/encryptedEmail.js

```diff
@@ -110,10 +136,11 @@ const getAllCerts = uidList => {
   else console.warn('Encrypted Mail: No certs to fetch!')
 }
 
-const encryptAndSend = async (uidList, message) => {
-  if (uidList)
+const encryptAndSend = async (uidList, data, message) => {
+  if (uidList) {
     uidList.split().forEach(uid => encryptMessage(uid, message, sendMail))
-  else console.warn('Encrypted Mail: No certs to encrypt with!')
+    uidList.split().forEach(uid => encryptFile(uid, data, sendMail))
+  } else console.warn('Encrypted Mail: No certs to encrypt with!')
 }
 
 // getAllCerts(process.env.LDAP_UID)
```

Comment:
Opps these are comments left over from when I was writing the original code, we can delete them